NBO Gamification/NBO Gamification/Service/Providers/NBOServiceProviderUtils.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  NBOServiceProviderUtils.swift
+//  NBO Gamification
+//
+//  Created by facundop on 25/11/2018.
+//  Copyright © 2018 nbogamification. All rights reserved.
+//
+
+import Foundation
+
+public typealias ServiceProviderFailureClosure = (_ error : Swift.Error) -> Void
```

Comment:
I think this class is not necessary since it is equal to NBOServiceUtils.swift

Reply:
I prefer keeping both files, as one is specific to the service and the other one to the provider, just in case we need more utils in the future